TrillSamples/HelloWorldFS/Program.fs

```diff
@@ -0,0 +1,67 @@
+﻿open System
+open System.Data.SqlTypes
+open System.Linq.Expressions
+open System.Reactive.Linq
+open Microsoft.StreamProcessing
+
+module Streamable =
+    open Microsoft.StreamProcessing
+
+    let ofObservable observable =
+        Streamable.ToStreamable(
+            observable, 
+            DisorderPolicy.Throw(), 
+            FlushPolicy.FlushOnPunctuation, 
+            PeriodicPunctuationPolicy.None(), 
+            OnCompletedPolicy.EndOfStream)
+            
+    let join mapping left right =
+        Streamable.Join(left, right, fun a b -> mapping a b)
+        
+    let multicast (mapping:IStreamable<'k,'a> -> IStreamable<'k,'b>) source =
+        Streamable.Multicast(source, mapping)
+        
+    let toStreamEventObservable (source:IStreamable<Empty,'a>) =
+        Streamable.ToStreamEventObservable(source)
+        
+    let alterEventLifetime (duration:int64) startTimeSelector source =
+        Streamable.AlterEventLifetime(source, (fun x -> startTimeSelector x), duration)
+        
+    let filter predicate source =
+        Streamable.Where(source, fun e -> predicate e)
+        
+    let map mapping source =
+        Streamable.Select(source, fun e -> mapping e)
+
+type SensorRange = { Time: int64; Low: int; High: int }
+type SensorReading = { Time: int64; Value: int }
+
+let historicData =
+    [ 0; 20; 15; 30; 45; 50; 30; 35; 60; 20 ]
+    |> Seq.mapi (fun i v -> { Time = (int64)(i + 1); Value = v } )
+
+let historicStream =
+    historicData
+    |> Observable.ToObservable
+    |> Observable.map (fun r -> StreamEvent.CreateInterval(r.Time, r.Time + 1L, r))
+    |> Streamable.ofObservable
```

Comment:
I would expect `Observable.toStreamable` here. Not a big deal, indeed.

Reply:
I agree. I think `toStreamable` looks nicer, but `Streamable` is a newer type. I am not in the least bit precious about this.

The intention here is to get some input on what sort of interface people would like for Trill.

I love to get some suggestions.

Reply:
Added